Issue 003: Shortcut does not work after power-up

boerge1 opened the pull request.

xfjx reviewed (changes requested):
Volume buttons still trigger shortcuts when short press is detected.

boerge1 commented:
@xfjx Hmm, I changed that also. Could you please check again and use really the latest version of the branch 'issue_003'. I removed the reaction on short press volume keys in the second commit.
